Tests/RoslynPluginGeneratorTests/AnalyzerPluginGeneratorTests.cs

```diff
@@ -25,6 +28,33 @@ private enum License { Required, NotRequired };
         private enum Node { Root, Child1, Child2, Grandchild1_1, Grandchild2_1, Grandchild2_2 };
 
         [TestMethod]
+        public void Generate_NoAnalyzersFoundInPackage_GenerateFails()
```

Comment:
I changed the behaviour of the generator slightly so it returns false if a jar isn't produced. This is a test for the new behaviour.